ble_chars.go

```diff
@@ -420,7 +420,7 @@ func setupBluetoothApplication(logger zerolog.Logger, coldBoot bool, vinLogger l
 	vinChar.OnRead(func(_ *service.Char, _ map[string]interface{}) (resp []byte, err error) {
 		defer func() {
 			if err != nil {
-				logger.Err(err).Ctx(context.WithValue(context.Background(), hooks.LogToMqtt, "true")).Msgf("error retrieving VIN via BLE: %s", err)
+				hooks.LogError(logger, err, "error retrieving VIN via BLE", hooks.WithThresholdWhenLogMqtt(1))
```

Comment:
this one we want to be logged in edge-logs... with this change is that the case?

Reply:
Yes, this one will continue be logged in edge-logs.
We want to do all error logging which will send over mqtt using hooks.LogError() func because not it is 2 ways to do it :
1. logger.Err(err).Ctx(context.WithValue(context.Background(), hooks.LogToMqtt, "true")
2. hooks.LogError(logger, err, "error retrieving VIN via BLE", hooks.WithThresholdWhenLogMqtt(1))

I want to get rid first, and use 2nd in all code